lib/interfaces/hapi.js

```diff
@@ -122,8 +122,8 @@ function hapiRegisterFunction(client, config, server, options, next) {
  */
 function handlerSetup(client, config) {
   var hapiPlugin = {register : hapiRegisterFunction.bind(null, client, config)};
-  var version = isPlainObject(config) && isString(config.version)
-                    ? config.version
+  var version = isPlainObject(config) && isString(config.version) ?
+                    config.version
```

Comment:
I meant that the `config.version` line should be `config.version : '0.0.0';`

Also, for readability, parentheses around `&&` expression.